stats/matrixify.py

```diff
@@ -92,12 +89,25 @@ def get_framework(path: str, rule: Dict[str, Any]) -> str:
     # if lang == hcl:
     return s[s.index(lang)+1].strip()
 
+# Reads 'cwe_to_metacategory.yml' to construct a map to convert a CWE to a metacategory
+def create_metacategory_map(path: str) -> Dict[str, str]:
+    cwe_mc_map = {} # {cwe, metacategory}
+
+    with open(path, 'r') as mc_map_file:
+        mc_map = yaml.safe_load(mc_map_file)
+
+        for mc in mc_map:
+            for cwe in mc_map[mc]:
+                cwe_mc_map[cwe] = mc
```

Comment:
super-nit: Can be replaced with the following one-liner
```suggestion
    return {cwe: mc for mc, cwes in mc_map for cwe in cwes}
```

Reply:
Ah neat. I like that it's more concise, but to be honest is harder for me to read and reason about :/ Will accept though 👍